src/JanusGraph.Net/Text.cs

```diff
@@ -0,0 +1,102 @@
+#region License
+
+/*
+ * Copyright 2018 JanusGraph.Net Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#endregion
+
+using Gremlin.Net.Process.Traversal;
+
+namespace JanusGraph.Net
+{
+    /// <summary>
+    ///     Provides text search predicates.
+    /// </summary>
+    public static class Text
+    {
+        /// <summary>
+        ///     Is true if (at least) one word inside the text string matches the query string.
+        /// </summary>
+        /// <param name="query">The query to search.</param>
+        /// <returns>The text predicate.</returns>
+        public static P TextContains(string query)
```

Comment:
Short expression could be used also: `public static P TextContains(string query) => new P("textContains", query);`